q_it_errors.rb

```diff
@@ -0,0 +1,18 @@
+
+# Note: Deriving QItStandardError Class from Ruby's StandardError Class 
+# and further deriving QIt Errors from QItStandardError.
+
+class QItStandardError < StandardError; end
```

Comment:
I think we don't need to go this granular with `NullSQSUrl` and `NullQueueName`, OR `InvalidQueueName` and `InvalidSQSUrl` exceptions. In case there are more arguments, you are not going to create new Classes for each one right? So have something like `NullArgument` or `InvalidArgument` exception that is applicable for all arguments, and reduce the number of error classes you create.

Less code is better code!